lib/grape/dsl/parameters.rb

```diff
@@ -238,6 +240,12 @@ def params(params)
         params = map_params(params, @element) if @element
         params
       end
+
+      private
+
+      def first_hash_key_or_param(parameter)
+        parameter.is_a?(Hash) ? parameter.keys[0] : parameter
```

Comment:
Maybe `.keys.first` is more idiomatic?

Reply:
Yes, it might.

The case is that I've just move the line that were in the `given` method `as is`.
Should I change the line then?

Reply:
It's nbd